### PR TITLE
update releasing nodes to include linux installers, homebrew and docker

### PR DESCRIPTION
**Pull request by gdams**

### CODE REVIEW

**Review by karianna — APPROVED** [review]
LGTM

File: RELEASING.md [hunk 1]
```diff
@@ -139,23 +139,27 @@ Here are the steps:
...
+1. [Mac only] Once the binaries are available on the website you need to update the Homebrew casks. There are 3 casks in total ([`temurin`](https://github.com/Homebrew/homebrew-cask/blob/master/Casks/temurin.rb) which always serves the latest version, [`temurin8`](https://github.com/Homebrew/homebrew-cask-versions/blob/master/Casks/temurin8.rb) and [`temurin11`](https://github.com/Homebrew/homebrew-cask-versions/blob/master/Casks/temurin11.rb)). An example PR can be found [here](https://github.com/Homebrew/homebrew-cask/pull/113581). If in doubt reach out to George (gdams) as he's a maintainer.
```
Comment (sxa) [inline RELEASING.md:156]: Clarify whether it needs to be three separate PRs against the three casks.
Also possibly provide a link to a JDK8 one alongside the JDK17 one as it's a "simpler" case without the change of `U` (which is also good as an example here of course!)

Comment (sxa) [reply RELEASING.md:156]: Perhaps add a note that saying that you can easily copy the the SHAs for inclusion in the PR from the temurin download website for [8](https://adoptium.net/releases.html?variant=openjdk8&jvmVariant=hotspot), [11](https://adoptium.net/releases.html?variant=openjdk11&jvmVariant=hotspot) and [17](https://adoptium.net/releases.html?variant=openjdk17&jvmVariant=hotspot)

File: RELEASING.md [hunk 2]
```diff
@@ -139,23 +139,27 @@ Here are the steps:
...
+  - **Debian** you need to locate the `rules` file for each version. This file contains the URLs and Checksums for each package ([example](https://github.com/adoptium/installer/blob/master/linuxNew/jdk/debian/src/main/packaging/temurin/11/debian/rules#L6-L15)).
+  - **RedHat/Suse** you need to modify the `temurin-<version>-jdk.spec` file for each version. The links/checksum links are all defined as `Source<number>` variables ([example](https://github.com/adoptium/installer/blob/master/linuxNew/jdk/redhat/src/main/packaging/temurin/11/temurin-11-jdk.spec#L128-L142)).
+1. [Docker] please see https://github.com/adoptium/containers#maintenance-of-dockerfiles
```
Comment (sxa) [inline RELEASING.md:160]: I think the formatting will need some fixup here as the two previous bullet points have caused this to show as step 1 again.

File: RELEASING.md [hunk 3]
```diff
@@ -139,23 +139,27 @@ Here are the steps:
...
+1. [Linux only] Once the binaries are available on the website you can begin updating the specfiles for the RPM/DEB files. There are 3 different type of linux installer ([debian](https://github.com/adoptium/installer/tree/master/linuxNew/jdk/debian/src/main/packaging/temurin), [redhat](https://github.com/adoptium/installer/tree/master/linuxNew/jdk/redhat/src/main/packaging/temurin) and [suse](https://github.com/adoptium/installer/tree/master/linuxNew/jdk/suse/src/main/packaging/temurin)) and all 3 need updating. Once these files have been updated the [adoptium-packages-linux-pipeline](https://ci.adoptopenjdk.net/job/adoptium-packages-linux-pipeline/) job needs to be kicked off making sure to tick the `UPLOAD` parameter.
```
Comment (sxa) [inline RELEASING.md:157]: Presumably it is valid to run the packages job without `UPLOAD` initially to use as a `dry-run` similar to the main publish job to verify that no errors occur? Perhaps worth explicitly making that suggestion.

**Review by sxa — CHANGES_REQUESTED** [review]
As per slack conversations can we also add some information on the use of the [aarch32 mirror job](https://ci.adoptopenjdk.net/view/git-mirrors/job/git-mirrors/job/git-hg-aarch32-jdk8u/) particularly in terms of the correct parameters that need to be used before running the build. Also are there any notes / protection we can put in place regarding [this failure](https://github.com/adoptium/mirror-scripts/issues/12)?

Also to close off https://github.com/adoptium/temurin-build/issues/2168 we should probably add a step "Make sure the GA tags have been mirrored to our repository and if not manually run <blah>"

**Comment by gdams** [comment]
> As per slack conversations can we also add some information on the use of the aarch32 mirror job particularly in terms of the correct parameters that need to be used before running the build. Also are there any notes / protection we can put in place regarding this failure?

@sxa that was added some time ago:

See the section in the README:
     - For HotSpot (arm32), the tag usually takes the form `jdk8u322-b04-aarch32-xxxxxxxx`
     - NOTE you need to set `overridePublishName` for arm32 to the actual OpenJDK tag (`jdk8u322-b04`)

**Comment by sxa** [comment]
>  @karianna requested a review from sxa yesterday 

Other than the replies about the need for the aarch32 mirror docs, my origin review comments still stand.

**Comment by sxa** [comment]
Also needs info about the changelog updates being required as that tripped me up when attempting the updates in this cycle.

File: RELEASING.md [hunk 4]
```diff
@@ -156,7 +156,11 @@ Here are the steps:
...
+1. [Linux only] Once the binaries are available on the website you can begin updating the specfiles for the RPM/DEB files. There are 3 different type of linux installer ([debian](https://github.com/adoptium/installer/tree/master/linuxNew/jdk/debian/src/main/packaging/temurin), [redhat](https://github.com/adoptium/installer/tree/master/linuxNew/jdk/redhat/src/main/packaging/temurin) and [suse](https://github.com/adoptium/installer/tree/master/linuxNew/jdk/suse/src/main/packaging/temurin)) and all 3 need updating. Once these files have been updated the [adoptium-packages-linux-pipeline](https://ci.adoptopenjdk.net/job/adoptium-packages-linux-pipeline/) job needs to be kicked off making sure to tick the `UPLOAD` parameter. It is suggested that you run the job with the `dry-run` parameter first to ensure that your parameters are correctly defined.
```
Comment (sxa) [inline RELEASING.md:160]: Ref my initial comment:

> Presumably it is valid to run the packages job without UPLOAD initially to use as a dry-run similar to the main publish job to verify that no errors occur?

I don't think the job currently has a parameter called `dry-run` (but we should possibly change the job to do so) so this needs to say "It is suggested that you run the job with `UPLOAD` unchecked first"

File: RELEASING.md [hunk 5]
```diff
@@ -156,7 +156,11 @@ Here are the steps:
...
+1. [Mac only] Once the binaries are available on the website you need to update the Homebrew casks. There are 3 casks in total ([`temurin`](https://github.com/Homebrew/homebrew-cask/blob/master/Casks/temurin.rb) which always serves the latest version, [`temurin8`](https://github.com/Homebrew/homebrew-cask-versions/blob/master/Casks/temurin8.rb) and [`temurin11`](https://github.com/Homebrew/homebrew-cask-versions/blob/master/Casks/temurin11.rb)). An example PR can be found [here](https://github.com/Homebrew/homebrew-cask/commit/4565865e9d7c3d3018ee8aa67803ea68c54dde86). Shamsums can be easilty copied from the website. The separate pull request is required for each version you update. If in doubt reach out to George (gdams) as he's a maintainer.
```
Comment (sxa) [inline RELEASING.md:159]:
```suggestion
1. [Mac only] Once the binaries are available on the website you need to update the Homebrew casks. There are 3 casks in total ([`temurin`](https://github.com/Homebrew/homebrew-cask/blob/master/Casks/temurin.rb) which always serves the latest version, [`temurin8`](https://github.com/Homebrew/homebrew-cask-versions/blob/master/Casks/temurin8.rb) and [`temurin11`](https://github.com/Homebrew/homebrew-cask-versions/blob/master/Casks/temurin11.rb)). An example PR can be found [here](https://github.com/Homebrew/homebrew-cask/commit/4565865e9d7c3d3018ee8aa67803ea68c54dde86). SHA sums can be easily copied from the website. The separate pull request is required for each version you update. If in doubt reach out to George (gdams) as he's a maintainer.
```

**Review by sxa — CHANGES_REQUESTED** [review]
A couple of remaining clarity / typo fixes required.

**Comment by sxa** [comment]
@gdams Is there anything blocking a response on the suggestions above?

**Pull request closed without merging**